### PR TITLE
hw-5(3)

### PR DESCRIPTION
**Pull request by glones**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+import java.util.List;
+
 public class FileWork {
+    private static final String LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        try {
+            List<String> preResult = new ArrayList<String>();
+            String file = Files.readString(Paths.get(fileName)).toLowerCase();
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:16]:
```suggestion
            String text = Files.readString(Paths.get(fileName)).toLowerCase();
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
+            if (file.length() > 0) {
+                for (String word : file.split("[^A-Za-zА-Яа-я]+")) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:18]:
```suggestion
                for (String word : file.split("\\W+")) {
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
+                    if (word.substring(0, 1).equals(LETTER)) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:19]: there is a method in class String that checks if a String starts with a certain value

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (glones) [reply src/main/java/core/basesyntax/FileWork.java]: done

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (glones) [reply src/main/java/core/basesyntax/FileWork.java]: done

File: src/main/java/core/basesyntax/FileWork.java (hunk 3 above)
Comment (glones) [reply src/main/java/core/basesyntax/FileWork.java]: done

**Pull request closed without merging**